### ISSUE TITLE
Latest CRC 1.33.1 installation steps and configuring the haproxy

### ISSUE THREAD

**Issue by rupeshad81** [code-ready/crc#2781]
### General information

  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / Hyper-V / hyperkit
  * Did you run `crc setup` before starting it (Yes/No)?
  * Running CRC on: Laptop / Baremetal-Server / VM

## CRC version
```bash
# Put `crc version` output here
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

[Uploading InstallingRedHatCRC_(CodeReadyContainers).docx…]()
Hi,

As per the CRC 1.33.1 , we need the proper steps to configure the crc on the linux.

on this link , we do not have updated CRC version 1.33.1 on the below link.

https://mirror.openshift.com/pub/openshift-v4/clients/crc/latest/

**Comment by praveenkumar** [comment]
@rupeshad81 Please check release guide https://github.com/code-ready/crc/releases/tag/v1.33.1 ,  the location to download latest version of crc updated there.

Also this issue doesn't provide any info around what is not working or any kind of logs so if this is only about download location then use that and close it otherwise update the issue to provide us info around what your tried and what is not working.

**Comment by rupeshad81** [comment]
Hi,


we are not able to download the setup.

root@eaas-crc ~]# wget -O- https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/crc/1.33.1/crc-linux-amd64.tar.xz | tar -xJf crc-linux-1.*-amd64/crc --strip-components 1
tar (child): crc-linux-1.*-amd64/crc: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
--2021-10-01 16:53:15--  https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/crc/1.33.1/crc-linux-amd64.tar.xz
Connecting to 10.232.233.70:8080... connected.
Proxy request sent, awaiting response... 307 Temporary Redirect
Location: https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/1.33.1/crc-linux-amd64.tar.xz [following]
--2021-10-01 16:53:15--  https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/1.33.1/crc-linux-amd64.tar.xz
Connecting to 10.232.233.70:8080... connected.
Proxy request sent, awaiting response... 302 Moved Temporarily
Location: https://access.cdn.redhat.com/content/origin/files/sha256/b6/b664ffffe50b98b6ad379aa159c78ee69cff4a0ae6e66414bdd7918afac535d1/crc-linux-amd64.tar.xz?_auth_=1633104157_90dd056dc636a7a3ecdf41a0ae40024b [following]
--2021-10-01 16:53:15--  https://access.cdn.redhat.com/content/origin/files/sha256/b6/b664ffffe50b98b6ad379aa159c78ee69cff4a0ae6e66414bdd7918afac535d1/crc-linux-amd64.tar.xz?_auth_=1633104157_90dd056dc636a7a3ecdf41a0ae40024b
Connecting to 10.232.233.70:8080... connected.
Proxy request sent, awaiting response... 200 OK
Length: 3151741112 (2.9G) [application/octet-stream]
Saving to: âSTDOUTâ

 0% [                                                                                                                                                   ] 0           --.-K/s   in 0s


Cannot write to â-â (Success).


Please can you provide the exact commands to download.

**Comment by rupeshad81** [comment]
Hi,

Yes it is working.

But unable to untar with the below command.

tar -Jx crc-linux-amd64.tar.xz/crc --strip-components 1

How can we do this step.

After this step we do the below step:

mv crc /usr/local/bin/

Please assist.


Thanks

**Comment by rupeshad81** [comment]
Hi,

One more update:
I was able to do untar and proceed.


Running the crc setup , it had given below errors:


[crc@eaas-crc ~]$ crc setup
CodeReady Containers is constantly improving and we would like to know more about usage (more details at https://developers.redhat.com/article/tool-data-collection)
Your preference can be changed manually if desired using 'crc config set consent-telemetry <yes/no>'
Would you like to contribute anonymous usage statistics? [y/N]: y
Thanks for helping us! You can disable telemetry with the command 'crc config set consent-telemetry no'.
INFO Checking if running as non-root
INFO Checking if running inside WSL2
INFO Checking if crc-admin-helper executable is cached
INFO Caching crc-admin-helper executable
INFO Using root access: Changing ownership of /home/crc/.crc/bin/crc-admin-helper-linux
INFO Using root access: Setting suid for /home/crc/.crc/bin/crc-admin-helper-linux
INFO Checking for obsolete admin-helper executable
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
INFO Checking if crc executable symlink exists
INFO Creating symlink for crc executable
INFO Checking if Virtualization is enabled
INFO Checking if KVM is enabled
INFO Checking if libvirt is installed
INFO Checking if user is part of libvirt group
INFO Checking if active user/process is currently part of the libvirt group
INFO Checking if libvirt daemon is running
INFO Checking if a supported libvirt version is installed
INFO Checking if crc-driver-libvirt is installed
INFO Installing crc-driver-libvirt
INFO Checking if systemd-networkd is running
INFO Checking if NetworkManager is installed
INFO Checking if NetworkManager service is running
INFO Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists
INFO Writing Network Manager config for crc
INFO Using root access: Writing NetworkManager configuration to /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf
INFO Using root access: Changing permissions for /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf to 644
INFO Using root access: Executing systemctl daemon-reload command
INFO Using root access: Executing systemctl reload NetworkManager
INFO Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists
INFO Writing dnsmasq config for crc
INFO Using root access: Writing NetworkManager configuration to /etc/NetworkManager/dnsmasq.d/crc.conf
INFO Using root access: Changing permissions for /etc/NetworkManager/dnsmasq.d/crc.conf to 644
INFO Using root access: Executing systemctl daemon-reload command
INFO Using root access: Executing systemctl reload NetworkManager
INFO Checking if libvirt 'crc' network is available
INFO Setting up libvirt 'crc' network
INFO Checking if libvirt 'crc' network is active
INFO Starting libvirt 'crc' network
INFO Checking if CRC bundle is extracted in '$HOME/.crc'
INFO Checking if /home/crc/.crc/cache/crc_libvirt_4.8.12.crcbundle exists
INFO Extracting bundle from the CRC executable
INFO Ensuring directory /home/crc/.crc/cache exists
INFO Extracting embedded bundle crc_libvirt_4.8.12.crcbundle to /home/crc/.crc/cache
INFO Uncompressing crc_libvirt_4.8.12.crcbundle
crc.qcow2: 11.90 GiB / 11.90 GiB [-------------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
oc: 117.41 MiB / 117.41 MiB [------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster
ERRO sending request - Post "https://api.segment.io/v1/batch": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
ERRO 2 messages dropped because they failed to be sent and the client was closed


What this 2 lines means?


Thanks

**Comment by praveenkumar** [comment]
> ERRO sending request - Post "https://api.segment.io/v1/batch": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
ERRO 2 messages dropped because they failed to be sent and the client was closed

If you have telemetry enabled then each action on crc command send to segment () https://developers.redhat.com/article/tool-data-collection. It might be the case that your proxy/firewall blocking the segment query hence this error. you can ignore it atm.

**Comment by rupeshad81** [comment]
Hi,

Ok Thanks.

I am proceeding further.

Thanks

**Comment by cfergeau** [comment]
You can disable telemetry in order to avoid seeing these errors

**Comment by praveenkumar** [comment]
Closing this issue because it was for not able to download the crc bits at first place. Please file new issue if you face any issue with crc.
